DQM/SiStripMonitorClient/src/SiStripInformationExtractor.cc

```diff
@@ -1,899 +0,0 @@
-#include "DQM/SiStripMonitorClient/interface/SiStripInformationExtractor.h"
```

Comment:
why is this file and the corresponding header being removed entirely?

Reply:
Honestly, I don't remember why I removed it.  Of course, it could be added back if needed.  As @Dr15Jones said above, I have no time to spend on CMS code, so I'm afraid someone else with have to restore it if that is necessary.

Reply:
I just checked, and nothing in CMSSW refers to SiStripInformationExtractor. It is therefore easier to remove then to continue fixing.